reorganizing readmes (#57)
risc0/bonsai-foundry-template: README.md

RISC0_DEV_MODE=false forge test
```

## Next Steps

> To build a basic Bonsai project, you'll need to make changes in two folders: 

Reviewer: Suggest "your application" to indicate that this is how to achieve their goal
```suggestion
To build your application, you'll need to make changes in two folders: 
```

        └── lib.rs                  // Built RISC Zero guest programs are compiled into here
```

> [methods]: /methods/README
> [contracts]: /contracts/README

Reviewer: These links are broken
```suggestion
[methods]: /methods
[contracts]: /contracts
```
